### PR TITLE
Add related projects section to `README`

### PR DESCRIPTION
**Pull request by joshwlambert**
This PR addresses #16 & #21 by adding a section to the package `README` on other R code available to simulate line list or contact data. 

Both a short description of each package has been given, and a table of features is provided after clicking a dropdown option. 

Please can reviewers check that: 

- the packages are described correctly
- have been assessed correctly in the feature table
- whether the criteria used in the table is adequate and fair
- whether there are other packages that are related to {simulist} but are missing from this PR

This PR also adds the `pkgdown/favicon` folder to the GitHub repository to prevent the pkgdown CI workflows failing. Fix comes from https://github.com/r-lib/pkgdown/issues/2553.

### CODE REVIEW

File: README.Rmd
Comment (CarmenTamayo) [inline README.Rmd:170]: so this package does the same as {simulist}? I think it'd be good to add a sentence to each of these highlighting what simulist brings to the table, for example user friendliness or the package being self contained and not relying on other packages- obviously in a subtle way without being negative about the other available packages

File: README.Rmd
Comment (CarmenTamayo) [inline README.Rmd:171]: maybe clarifying what surveys here?

**Review by CarmenTamayo — APPROVED** [review]
I've suggested a couple of changes to the Readme, mainly to highlight a bit more what simulist is doing that sets it apart from other packages, but apart from that I'm happy for the changes to be merged.

File: README.Rmd
Comment (CarmenTamayo) [inline README.Rmd:163]: In the original simulist project idea I had also listed [ringbp](https://github.com/epiforecasts/ringbp) as an example of a related project since you can also use that package to simulate outbreak data with contact tracing- this however doesn't give you a linelist, although the output could be used to generate a linelist as well, this could be mentioned in this section to highlight why simulist is useful

File: README.Rmd
Comment (CarmenTamayo) [inline README.Rmd:171]:
```suggestion
- [`{epidict}`](https://github.com/R4EPI/epidict) is a package that can be used to simulate outbreak data, including linelists, in a DHIS2 format, and survey data that mimics the format by Kobo, using the function `gen_data()`. In addition, MSF outbreak data are available in this package as data dictionaries for Acute Jaundice Syndrome, Cholera, Measles and Meningitis, accessible through the function `msf_dict()`.
```

File: README.Rmd
Comment (CarmenTamayo) [inline README.Rmd:185]: Columns that I think need a footnote: parameterised with epi distributions, explicit population size, unit testing

File: README.Rmd
Comment (CarmenTamayo) [reply README.Rmd:163]: I was going to add them both but while I was doing so I started to question whether they're actually relevant enough to add them to the table since they're not able to simulate linelists, which is the main point of simulist... what do you think? maybe they could be mentioned in a separate bullet point to say that there are other packages that might do stuff relevant for outbreak simulation but not necessarily provide tailored linelists to the user

**Review by CarmenTamayo — COMMENTED** [review]
I used the other packages and I believe the assessment is correct with what's included on the table, I have left a few more comments with suggestions or questions for @joshwlambert

File: README.Rmd
Comment (joshwlambert) [reply README.Rmd:170]: I'm not sure that going into detail on the differences between the packages is required for the `README`. This section is more of a guide to showcase which other packages are available with a brief description. In my opinion it is up to the user to decide what is the appropriate tool for their use case. 

As you mentioned, it's difficult to advocate for {simulist} versus the other listed packages without being negative about them. 

Hopefully the table of features will provide a high-level guide to which tool is appropriate to use (e.g. showing which packages are no longer maintained).

File: README.Rmd
Comment (joshwlambert) [reply README.Rmd:172]: > Is the list's justification to compare simulist to existing projects to showcase what simulist can do that others can't or to act as a resource so users can access other packages for those cases where using simulist is not an option?

I would say definitely the latter. As an open source project, {simulist} doesn't want to claim a monopoly of functionality over every other package that is related or overlaps with {simulist}. This is just a guide in case the user wants to work out which tools can help them to accomplish their task. As part of this the feature table will highlight what {simulist} can do that the other packages can't (or vice versa) but it is not to show what other packages cannot do.

> most importantly it's a deprecated package- should we even include it in this list?

For now I think it's best to include, however, if over time we think that we should remove deprecated packages from the list then we can do. Arguably, packages that are not maintained and haven't been developed in several years could also be termed deprecated, it is just the maintainer has not explicitly tagged the package as such.

File: README.Rmd
Comment (joshwlambert) [reply README.Rmd:183]: I will leave the "Simulates contacts" field as ✅ for {LLsim} as the contact information is returned and can be used to generate an `<epicontacts>` object, see https://github.com/jrcpulliam/LLsim/blob/master/vignettes/plagueJoshi.Rmd. I see you point about the separate `<data.frame>`, and I prefer this approach in {simulist}, but I still think the {LLsim} approach warrants a ✅.

File: README.Rmd
Comment (joshwlambert) [reply README.Rmd:163]: Thanks for the suggestion, I've added a few bullet points below what was already added to the Related projects section linking to {outbreak}, {ringbp} and {epichains} in 8e4370342e0fefb4225667581fcf04176fca350a.

File: README.Rmd
Comment (joshwlambert) [reply README.Rmd]: I've added some text on the 2-step simulation required by {LLsim} in a3cbb0dff03e44f9d68ba39fb5c085e2afdf9e5d.

File: README.Rmd
Comment (joshwlambert) [reply README.Rmd:185]: I've added footnotes for all of these fields in 2ff93f0e16b283d30ff09a19c5436516247bcc7d.

**Comment by joshwlambert** [comment]
Thanks for all the feedback on this PR. I will merge the changes once CI checks pass. The related projects section of the `README` will evolve as new packages are developed or existing packages gain new features or become deprecated. Therefore, if anyone sees new packages or changes to existing information on packages please make an issue or a PR and these can be updated.

**Pull request merged**
